generators/gulp/templates/gulpfile.js

```diff
@@ -34,6 +35,13 @@ gulp.task('nsp', function (cb) {
   nsp({package: path.resolve('package.json')}, cb);
 });
 
+gulp.task('lec', function () {
+  return gulp.src('**/*')
```

Comment:
Actually, as this is in pre-commit, it doesn't matter that much. But it'd still be better to only have cli file modified for windows users.